pkg/config.go

```diff
@@ -0,0 +1,31 @@
+package pkg
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"gopkg.in/yaml.v2"
+)
+
+// ReadConfig : Read config and call CheckConfig
+func ReadConfig(configfile string) Config {
```

Comment:
This should probably be split into a ParseConfig and RunChecks or similar method